.travis.yml

```diff
@@ -13,10 +13,11 @@ after_success:
 matrix:
   fast_finish: true
   include:
-  - python: '3.8'
+  - name: lint
+    python: '3.8'
     install: pip install flake8
     script: make lint
-  include:
-  - python: '3.8'
+  - name: format
+    python: '3.8'
```

Comment:
Thanks for fixing this. I have the hardest time making sense of Travis' config format.

Reply:
Yep. (I'm in the process of switching over to GitHub actions in some of my other projects.)

Reply:
Yea, we should discuss which CI we want to use. I've had mixed experiences with GH Actions. They haven't been as reliable as other CI's I've used. Though I like that they are easy to use on other people's projects. 😁 My default these days is still CircleCI, though it has its own downsides.

That Travis builds aren't showing up in the PRs is a bit aggravating. 😭

Reply:
About once a year, something mysteriously stops working with Travis-CI integration, and I end up having to disconnect and reconnect it to my repos. (To be fair, I think this is partly GH's fault, as they've changed how tools are supposed to integrate over the years.)